### ISSUE TITLE
[MiniLLM] mismatch between formula and implementation of the single-step loss？

### ISSUE THREAD

**Issue by YifanHao** [microsoft/LMOps#255]
Hello, I was trying to have a better understanding of your method, but I got a little confused about the single-step part in the loss. 

[ppo_loss](https://github.com/microsoft/LMOps/blob/7bc6eed314998a3b58f706bc9049a99f451efaa5/minillm/minillm/losses.py#L122)
In your implementation, I saw you compute `ppo_loss = pg_loss + single_step_reg_loss`. It seems the `single_step_reg_loss` corresponds to the L_single part in the formula, and `pg_loss` is the L^norm_Long part. 
![image](https://github.com/user-attachments/assets/af4f4bc7-dd65-47c3-ab1d-2240a3d0eaa0)
Then, the `single_step_reg_loss` is computed by `_reg_loss`, which seems like an average of step-wise reversed KLD on sampled sequence, and reward is not involved in the computation. 

![image](https://github.com/user-attachments/assets/3af880b2-6e0b-4be6-b7a6-bfecad2d2741)
![image](https://github.com/user-attachments/assets/674ed9d5-e8da-4154-9410-40e202534458)
However, according to fomula above, the single-step part should be an weighted average of step-wise rKLD, with reward at each step as the weight. 

So why is the reward weight missing in the implemetation？Am I missing something, and could you help me clarify it? Thanks for your help!

**Comment by t1101675** [comment]
Thanks for your question! We find removing $w_t$ from the single-step part will make the training converges splightly faster in our pilot studies. Sorry for the confusion. We will clarify this in our code and our paper in the next version.

**Comment by YifanHao** [comment]
Thanks for your reply!
